pages/download.py

```diff
@@ -87,13 +87,26 @@ def __init__(self):
         self.title = _("Download")
         self.slug = "download"
     def get_content(self):
+        download_intro = md(_("""
+To access Freenet, you first need to install the main application. Freenet will run in the background and you can use
+your browser to change settings and access content. There are other applications that you can install
+at a later time to add more functionality.
```

Comment:
Please avoid breaking/wrapping lines at arbitrary points. Doing so introduces unnecessary complexity in the Git history and might make use of translation memories slightly harder.